ldk/csharp/OliveHelpsLDK/Example/Program.cs

```diff
@@ -29,6 +30,11 @@ class Program
         public static void Main(string[] args)
         {
             ILogger logger = new Logger("csharp-clipboard-example");
+            TaskScheduler.UnobservedTaskException += (sender, eventArgs) =>
+            {
+                logger.Error("Unhandled Task Exception",
```

Comment:
I see what you're saying now - I'll poke at pulling this kind of thing up into the interceptor stuff.